src/Dotnet.Script.Core/Scaffolder.cs

```diff
@@ -141,7 +141,7 @@ private void CreateLaunchConfiguration(string currentWorkingDirectory)
             _scriptConsole.WriteNormal("Creating VS Code launch configuration file");
             string pathToLaunchFile = Path.Combine(vsCodeDirectory, "launch.json");
             string installLocation = _scriptEnvironment.InstallLocation;
-            bool isInstalledAsGlobalTool = installLocation.Contains(".dotnet/tools", StringComparison.OrdinalIgnoreCase);
+            bool isInstalledAsGlobalTool = installLocation.Contains($".dotnet{Path.DirectorySeparatorChar}tools", StringComparison.OrdinalIgnoreCase);
```

Comment:
Better to use `Path.Combine(".dotnet", "tools")`.

Reply:
internally it uses `DirectorySeparatorChar` too so it's a matter of preference